src/components/Tutor/BaseURL.js

```diff
@@ -0,0 +1,68 @@
+function isValid(val) {
+  return val !== "";
+}
+
+export class BaseURL {
+  URL = `${process.env.REACT_APP_MERAKI_URL}/volunteers`;
+
+  /*Setting date range */
+  setDates(fromStart, toEnd) {
+    this.URL = this.URL.concat(`?from=${fromStart}&to=${toEnd}`);
+  }
+
+  /*Filters and Search */
+  /*  Total varialbles = n = 3
+        Number of cases = nC3 + nC2 + nC1 = 1 + 3 + 3 = 7
```

Comment:
Probably easier to get the number of possibilities by thinking `n` filters, `2` choices for each, which would give you `2**n`. But since you're excluding the no filter choice, there's one less choice so you're using `2**n - 1`.